File: codeforces/random/A/CF_236A.java
import java.util.Arrays;
import java.util.Scanner;

public class CF_236A {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        char[] s = sc.next().toCharArray();

        int l = s.length;
        Arrays.sort(s);
        System.out.println(s);
        System.out.println(l);



        int count = 1;
        for(int i=0; i<=l-1; i++) {
            if(s[i] != s[i+1]) {
                count++;
            }
        }

        if(count%2 == 0) {
            System.out.println("CHAT WITH HER!");
        } else {
            System.out.println("IGNORE HIM!");
        }

    }
}

File: codeforces/random/A/CF_236A.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
	char str[101];
 
	cin >> str;
	int l  = strlen(str);
	sort(str, str+l);
	
    int count =0;
    for(int i=0; i<l; i++) {
        if(str[i] != str[i+1]) {
            count++;
        }
    }

    if(count%2==0) {
        cout << "CHAT WITH HER!" << endl;
    } else {
        cout<<"IGNORE HIM!" << endl;
    }
}
